lib/opentelemetry/types.ts

```diff
@@ -2,24 +2,46 @@ import { Strategy } from "unstructured-client/sdk/models/shared";
 import { z } from "zod";
 
 export const ExternalServiceName = {
+	Anthropic: "anthropic",
 	Firecrawl: "firecrawl",
+	Google: "google",
 	OpenAI: "openai",
 	Tavily: "tavily",
 	Unstructured: "unstructured",
 	VercelBlob: "vercel_blob",
 } as const;
 
-export type ExternalServiceName = // Name of the service to which agent requests
-	(typeof ExternalServiceName)[keyof typeof ExternalServiceName];
+export const UnimplementedServiceName = {
+	TODO: "todo", // for type safety
```

Comment:
```suggestion
	Unknown: "unknown", // for type safety
```

Reply:
nits: 

The text reads "TODO," but I don't think there's actually anything to be done here. I think something like "Unimplemented" or "Unknown" would be more appropriate.  

Incidentally, "Unknown" was a word suggested by Copilot.

https://github.com/giselles-ai/giselle/pull/221/files#diff-49a97b0518ec7e083f544888dcd1be31d3b81c5f68775c2c465041ae9def9fdbR58 is also "unknown".